integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -9,67 +9,53 @@
 # See the NOTICE file distributed with this work for information regarding copyright ownership.
 #
 
+# Do NOT modify
 fullnameOverride: alluxio
 
+
+## Common ##
+
+# Docker Image
 image: alluxio/alluxio
 imageTag: 2.1.0-SNAPSHOT
 imagePullPolicy: IfNotPresent
 
-# Set the tieredstore
-tieredstore:
-  levels:
-  - alias: MEM
-    level: 0
-    path: /dev/shm
-    type: emptyDir
-    quota: 1G
-    high: 0.95
-    low: 0.7
-
-# set the uid, gid and fsGroup
+# Security Context
 user: 1000
 group: 1000
 fsGroup: 1000
 
-# The domain hostPath for uuid mode in short Circuit
-domainHostPath: "/tmp/alluxio-domain"
-
-# properties for all the components
+# Site properties for all the components
 properties:
   # alluxio.user.metrics.collection.enabled: 'true'
   alluxio.security.stale.channel.purge.interval: 365d
 
 # Recommended JVM Heap options for running in Docker
 # Ref: https://developers.redhat.com/blog/2017/03/14/java-inside-docker/
+# These JVM options are common to all Alluxio services
 # jvmOptions: "-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -XX:MaxRAMFraction=2 "
 
-fuse:
-  image: alluxio/alluxio-fuse
-  imageTag: 2.1.0-SNAPSHOT
-  imagePullPolicy: IfNotPresent
-  # Customize the MaxDirectMemorySize
-  jvmOptions: " -XX:MaxDirectMemorySize=2g "
-  # shortCircuitPolicy can be local or uuid,
-  # local means the cache directory is in the same mount namespace,
-  # uuid means interact with domain socket
-  shortCircuitPolicy: uuid
-  hostNetwork: true
-  dnsPolicy: ClusterFirstWithHostNet
-  args:
-    - fuse
-    - --fuse-opts=allow_other
+# Mount Persistent Volumes to all components
+# mounts:
+# - name: <persistentVolume claimName>
+#   path: <mountPath>
+
+# Use labels to run Alluxio on a subset of the K8s nodes
+# nodeSelector: {}
+
+## Master ##
 
 master:
   count: 1 # For multiMaster mode increase this to >1
-  replicas: 1
-  args:
-    - master
-    - -no-format
-  # properties for the master components
+  replicas: 1 # Do NOT modify
```

Comment:
Do we need a reason for "Do NOT modify" comments?

Reply:
count vs replicas was confusing to me. so i felt like i needed to mention which one to modify. ideally i would remove it from `values.yaml` altogether

Reply:
Yea that's true. Maybe mention in the comment that this controls the number of replicas in the StatefulSet and mention in the comment for `count` that `count` controls the number of masters?